vhdbuilder/packer/components.json

```diff
@@ -122,15 +125,18 @@
       ],
       "multiArchVersions": [
         "v1.0.0.2",
-        "v1.0.0.3"
+        "v1.0.0.3",
+        "v1.1.1.3",
```

Comment:
curious why v1.1.1.3 and v1.1.1.4 are both in use.

Reply:
I think v1.1.1.3 is in use but not preloaded,
while v1.1.1.4 is latest patch version